src/containers/ui/Tooltip.js

```diff
@@ -0,0 +1,12 @@
+import { connect } from 'react-redux';
+import Tooltip from 'components/ui/Tooltip';
+import { toggleTooltip, tooltipSetPosition } from 'actions/tooltip';
+
+const mapStateToProps = ({ tooltip }) => ({
+  tooltip
+});
+
+export default connect(mapStateToProps, {
+  toggleTooltip,
+  tooltipSetPosition
```

Comment:
Let's use always the verb first. => setTooltipPosition. Only for actions. What do you think?